lib/attr_vault/cryptor.rb

```diff
@@ -6,7 +6,7 @@ module Cryptor
     def self.encrypt(value, key)
       return value if value.nil? || value.empty?
 
-      secret = AttrVault::Secret.new(key)
+      secret = AttrVault::Secret.new(key, aead: false)
```

Comment:
I'm kind of finding myself wishing that Secret and Key were more unified somehow, and could select a Cryptor instance, but am not sure about how much we want to reorganize the program when a couple of branches may give this another five to ten years of treads.